bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonDeviceHandler.java

```diff
@@ -373,7 +375,9 @@ public void initialize() {
                 });
 
                 if (ThingStatus.ONLINE == bridge.getStatus()) {
-                    updateStatus(ThingStatus.ONLINE);
+                    if (deviceLinked) {
+                        updateStatus(ThingStatus.ONLINE);
+                    }
```

Comment:
Unfortunately, that will not work since the network handler might still be in the process of downloading the database and parsing it. `if (deviceLinked)` only comes into play when a user manually adds a device from the UI and enters an address that isn't in the modem database.